.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.0.5
+current_version = 1.0.7
```

Comment:
The version is wrong - should be really `1.0.5`. If you want to create a new PyPI release, I'd do it after merging this PR.